go/pserver/cclient/test/dump_optimizer_proto.py

```diff
@@ -0,0 +1,13 @@
+import OptimizerConfig_pb2 as pb
```

Comment:
What is this file for? If we only need the generated proto file, maybe we can just check in the protofile under `testdata` folder.